config/example/calib/process_modifier_test.calib

```diff
@@ -0,0 +1,3 @@
+# X Y
```

Comment:
Please do not add a new folder _calib_ just for the example. Rather define a 1:1 interpolation as default value (no change in data) and do not enforce the load from file.